Developer-Guide.md

```diff
@@ -353,6 +355,23 @@ $ popd
 $ rm -rf "${tmpdir}"
 ```
 
+# Install Hypervisor
+
+`qemu-lite` will be installed automatically if you're using the [OBS packaging](https://github.com/kata-containers/documentation/tree/master/install)
+
```

Comment:
That's more or less what `install_qemu.sh` below is doing.

Reply:
Needs a space here: *...methods, you...*

Reply:
@marcov - correct. I thought it would be useful to "spell it out" like this though as it would allow users to start with a known good config for Kata (`kata.cfg`) which they could then adjust to their needs before actually configuring.